contributed/hackurdc2_utils/units.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, division, print_function
+import numpy as np
+
```

Comment:
Are these utils that we could get from somewhere else, eg. CCL?

Reply:
I'm not sure (probably), but I thought these conversions were easy enough to understand and lightweight.